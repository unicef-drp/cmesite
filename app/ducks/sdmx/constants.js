export const LOAD_STRUCTURE = 'ducks/sdmx/LOAD_STRUCTURE';
export const LOAD_STRUCTURE_SUCCESS = 'ducks/sdmx/LOAD_STRUCTURE_SUCCESS';
export const LOAD_STRUCTURE_ERROR = 'ducks/sdmx/LOAD_STRUCTURE_ERROR';
