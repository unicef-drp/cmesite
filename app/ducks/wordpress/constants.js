export const LOAD_POSTS = 'ducks/wordpress/LOAD_POSTS';
export const LOAD_POSTS_SUCCESS = 'ducks/wordpress/LOAD_POSTS_SUCCESS';
export const LOAD_POSTS_ERROR = 'ducks/wordpress/LOAD_POSTS_ERROR';
