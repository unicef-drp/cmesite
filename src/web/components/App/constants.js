export const PREFIX = '@@drsa-web-app/';

export const DEFAULT_LOCALE = 'en';

export const PROD_ENV = 'prd';
