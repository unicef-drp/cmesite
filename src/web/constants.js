export const REF_AREA = 'REF_AREA';
export const INDICATOR = 'INDICATOR';
export const SEX = 'SEX';
export const TIME_PERIOD = 'TIME_PERIOD';
export const RELEVANT_DIMENSIONS = [REF_AREA, INDICATOR, SEX];
export const RELEVANT_DIMENSIONS_DEFAULTS = new Set(['MRY0T4', '_T']);

export const ESTIMATE = 'ESTIMATE';
export const INCLUDED = 'INCLUDED';
export const EXCLUDED = 'EXCLUDED';

export const ESTIMATE_TYPE = { sdmxId: 'SERIES_NAME', sdmxValue: '269', id: ESTIMATE, value: true };
export const TYPES = [
  ESTIMATE_TYPE,
  { sdmxId: 'OBS_STATUS', sdmxValue: 'IN', id: INCLUDED, value: true },
  { sdmxId: 'OBS_STATUS', sdmxValue: 'EX', id: EXCLUDED, value: false },
];

export const Z = 'SERIES_NAME';
export const X = TIME_PERIOD;
export const Y0 = 'LOWER_BOUND';
export const Y1 = 'UPPER_BOUND';
