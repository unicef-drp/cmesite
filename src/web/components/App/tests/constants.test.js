import * as Constants from '../constants';

describe('app | containers |Â App | constants', () => {
  it('should be defined', () => {
    expect(Constants).toBeDefined();
  });

  it('should match snapshot', () => {
    expect(Constants).toMatchSnapshot();
  });
});
