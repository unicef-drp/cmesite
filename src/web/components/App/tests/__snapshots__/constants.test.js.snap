// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`app | containers |Â App | constants should match snapshot 1`] = `
Object {
  "DEFAULT_LOCALE": "en",
  "PREFIX": "@@drsa-web-app/",
  "PROD_ENV": "prd",
}
`;
